var/spack/repos/builtin/packages/py-pcircle/package.py

```diff
@@ -0,0 +1,45 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyPcircle(PythonPackage):
+    """ A suite of parallel file system tools."""
+    homepage = "http://github.com/olcf/pcircle"
+    version('develop', git='http://github.com/olcf/pcircle.git',
+            branch='dev')
+
+    import_modules = ['openmpi', 'python']
+
+    phases = ['install']
+
+    depends_on('python@2.7.14:')
+    depends_on('py-mpi4py')
+    depends_on('py-scandir')
+    depends_on('py-xattr')
+    depends_on('py-numpy')
+    depends_on('py-bitarray')
+    depends_on('py-cffi', type='build')
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
Missing a dependency on `py-lru-dict`. Can you also add the specific version requirements?
```python
requires = [                                                                    
    'cffi>=1.2.1',                                                              
    'mpi4py>=1.3.1',                                                            
    'lru-dict>=1.1.1',                                                          
    'xattr',                                                                    
    'scandir>=1.1',                                                             
    'numpy>=1.9.2',                                                             
    'bitarray>=0.8.1'                                                           
]             
```
Also, all of these (except `py-cffi`) should be `type=('build', 'run')` as they don't need to be linked.